liquibase-core/src/test/java/liquibase/sqlgenerator/core/CreateViewGeneratorTest.java

```diff
@@ -51,6 +62,139 @@ public void generateSql_mssqlWithSetStatements_addsSetInProperOrder() {
         assertEquals("SET QUOTED_IDENTIFIER ON", sqls[5].toSql());
     }
 
+
+    @Test
+    public void cleanUpSqlString_noTrailingComment_slashCleanUpDone() {
```

Comment:
All the test cases look good, but could you please reformat all of them a bit so they are consistent with our conventions where we add new line between arrange act and assert? (https://datical.atlassian.net/wiki/spaces/DEV/pages/3542057033/Enterprise+team+code+conventions)
e.g. for this test case it will be 
```
        String initialSql = "   CREATE VIEW SOME_VIEW AS SELECT * FROM SOME_TABLE" + LSP + "     /";

        StringClauses viewDefinition = SqlParser.parse(initialSql, true, true);

        String expectedCleanSql = "   CREATE VIEW SOME_VIEW AS SELECT * FROM SOME_TABLE" + LSP + "     ";
        assertEquals(expectedCleanSql, generator.getViewDefinition(viewDefinition, database));

```

Reply:
done